test/katex-spec.js

```diff
@@ -3066,6 +3066,21 @@ describe("A macro expander", function() {
             .toParseLike`11\sqrt[2]{2}11`;
     });
 
+    it("array cells generate groups", () => {
+        expect`\def\x{1}\begin{matrix}\x&\def\x{2}\x&\x\end{matrix}`
+            .toParseLike`\begin{matrix}1&2&1\end{matrix}`;
```

Comment:
Interesting... so before this change, the output would've been `\begin{matrix}1&2&2\end{matrix}`?

Reply:
Yes, exactly!

![image](https://user-images.githubusercontent.com/2218736/58562070-8122f300-81f6-11e9-8925-a5ab4eb5c603.png)